desktop/settings/mac-settings.md

```diff
@@ -0,0 +1,193 @@
+
+- **Send usage statistics**. Select so Docker Desktop sends diagnostics, crash reports, and usage data. This information helps Docker improve and troubleshoot the application. Clear the check box to opt out.
+
+- **Show weekly tips**. Select to display useful advice and suggestions about using Docker.
+
+- **Open Docker Desktop dashboard at startup**. Select to automatically open Docker Dashboard when starting Docker Desktop.
+
+- **Use Docker Compose V2**. Select to enable the `docker-compose` command to use Docker Compose V2. For more information, see [Docker Compose V2](../../compose/index.md#compose-v2-and-the-new-docker-compose-command).
+
+## Resources
+
+The **Resources** tab allows you to configure CPU, memory, disk, proxies, network, and other resources.
+
+### Advanced
+
+On the **Advanced** tab, you can limit resources available to Docker.
+
+Advanced settings are:
+
+- **CPUs**. By default, Docker Desktop is set to use half the number of processors
+available on the host machine. To increase processing power, set this to a
+higher number; to decrease, lower the number.
+
+- **Memory**. By default, Docker Desktop is set to use `2` GB runtime memory,
+allocated from the total available memory on your Mac. To increase the RAM, set this to a higher number. To decrease it, lower the number.
+
+- **Swap**. Configure swap file size as needed. The default is 1 GB.
+
+- **Disk image size**. Specify the size of the disk image.
+
+- **Disk image location**. Specify the location of the Linux volume where containers and images are stored.
+
+You can also move the disk image to a different location. If you attempt to move a disk image to a location that already has one, you are asked if you want to use the existing image or replace it.
+
+#### File sharing
+
+Use File sharing to allow local directories on the Mac to be shared with Linux containers.
+This is especially useful for
+editing source code in an IDE on the host while running and testing the code in a container.
+By default the `/Users`, `/Volume`, `/private`, `/tmp` and `/var/folders` directory are shared. If your project is outside this directory then it must be added
+to the list. Otherwise you may get `Mounts denied` or `cannot start service` errors at runtime.
+
+File share settings are:
+
+- **Add a Directory**. Click `+` and navigate to the directory you want to add.
+
+- **Remove a Directory**. Click `-` next to the directory you want to remove
+
+- **Apply & Restart** makes the directory available to containers using Docker's
+  bind mount (`-v`) feature.
+
+> Tips on shared folders, permissions, and volume mounts
+>
+ * Share only the directories that you need with the container. File sharing
+ introduces overhead as any changes to the files on the host need to be notified
+ to the Linux VM. Sharing too many files can lead to high CPU load and slow
+ filesystem performance.
+>
```

Comment:
This note looks to be formatted a bit odd (just the source code, it renders fine as HTML). I think we generally;

- repeat the `>` on each line
- use a hanging indent for bullet lists to make them look more like a bullet list also when viewing the source


```markdown
> Tips on shared folders, permissions, and volume mounts
>
> * Share only the directories that you need with the container. File sharing
>   introduces overhead as any changes to the files on the host need to be notified
>   to the Linux VM. Sharing too many files can lead to high CPU load and slow
>   filesystem performance.
...
```